module/mailnews/newsread.c

```diff
@@ -389,8 +411,137 @@ Bool DateFromSeconds(long seconds, char *str)
    if (t == NULL)
       return False;
 
-   sprintf(str, "%s %s %.2ld, %.4ld %.2ld:%.2ld", 
+   snprintf(str, 50, "%s %s %.2ld, %.4ld %.2ld:%.2ld", 
 	   GetString(hInst, weekdays[t->tm_wday]), GetString(hInst, months[t->tm_mon]), 
 	   t->tm_mday, t->tm_year + 1900, t->tm_hour, t->tm_min);
    return True;
 }
+
+void OnColumnClick(LPNMLISTVIEW pLVInfo)
+{
+    int nSortColumn = pLVInfo->iSubItem;
+    BOOL bSortAscending = TRUE;
```

Comment:
Use bool (C++ built-in type) instead of BOOL (horrible Windows-ism).

Reply:
_All_ instances of `BOOL` or `Bool` to `bool`? There were a few not highlighted in your comments.

Reply:
Use BOOL where you have to, such as parameters to Windows functions, bool everywhere else.  Bool and BOOL are from a time before C++ existed.